interface/patient_file/encounter/forms.php

```diff
@@ -30,38 +30,84 @@
 <html>
 
 <head>
-<?php html_header_show();?>
+<?php
+html_header_show();
+
+/**
+ * Generate a script tag based on array of elements
+ *
+ * $options = [
+ *     [
+ *         'src' => 'Source path of the file to include',
+ *         'basePath' => 'Defaults to $GLOBALS['assets_static_relative']',
+ *         'atts' => ['Array of attributes. Not currently supported, future use'],
+ *     ]];
+ *
+ * Options can also be an array of strings in which case each string represents the path (basePath will be prepended),
+ * in which case:
+ *
+ * $options = ['jquery-latest/jquery.js', 'bootstrap-latest/bootstrap.js'];
+ *
+ * @param $options array Array of items
+ * @return string
+ */
+function generateScriptElements($options)
+{
+    $template = '<script src="{src}"></script>';
+    $return = [];
+    $basePath = $GLOBALS['assets_static_relative'] . '/';
+    foreach ($options as $element) {
+        if (is_array($element)) {
+            if (array_key_exists('basePath', $element) && $element['basePath'] !== false) {
+                $basePath = $element['basePath'];
+            } else if (array_key_exists('basePath', $element) && $element['basePath'] === false) {
+                // If basePath gets passed in but is false, ensure it's not appended
+                $basePath = "";
+            }
+            $str = str_replace("{src}", $basePath . $element['src'], $template);
+        } else {
+            $str = str_replace("{src}", $element, $template);
+        }
+        $return[] = $str;
+    }
+    return implode("\n", $return);
+}
+
+$libraryDir = $GLOBALS['webroot'] . '/library/';
+$scripts = [
+    ['src' => 'jquery-min-1-7-2/index.js'],
+    ['basePath' => $libraryDir, 'src' => 'dialog.js?v=' . $v_js_includes],
+    ['basePath' => $libraryDir, 'src' => 'textformat.js'],
+    ['basePath' => $libraryDir, 'src' => 'dynarch_calendar.js'],
+    ['basePath' => $libraryDir, 'src' => 'dynarch_calendar_setup.js'],
+    ['basePath' => $libraryDir, 'src' => 'js/common.js'],
+    ['basePath' => $libraryDir, 'src' => 'js/fancybox-1.3.4/jquery.fancybox-1.3.4.js'],
+    ['basePath' => $libraryDir, 'src' => 'ESign/js/jquery.esign.js'],
+    ['basePath' => $libraryDir, 'src' => 'openflashchart/js/json/json2.js'],
+    ['basePath' => $libraryDir, 'src' => 'openflashchart/js/swfobject.js'],
+];
+?>
```

Comment:
I would shelve this stuff for now until we have a good strategy in place for this. Note there is an ongoing standard template project going on where we are learning a lot about the codebase as we standardize the headers. When this is completed and the algorithm of what we need to support is clear, then we can formulate a nice modern approach to encapsulate and centralize this.